### DISCUSSION TITLE
Which is best between redux and globalstate for auth?

### DISCUSSION THREAD

**Discussion by spyhunter128** [community/community#65990] [Projects and Issues]
### Select Topic Area

Question

### Feature Area

Issues

### Body

Hello, everyone

I have to implement auth part for website.
website is made by next.js
which is best between redux and global state?

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by cohen1280** [comment]
global state is best
global state is more simple than redux
auth part is not complicated, so I think, global state is best for your situation.

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
